src/components/Navbar.tsx

```diff
@@ -11,7 +11,7 @@ import { useSession } from "next-auth/react";
 
 const navigation = [{ name: "Home", href: "/app", current: true }];
 
-const profileLinks = [{ name: "Settings", onClick: () => {} }, { name: "Sign out", onClick: () => signOut() },];
+const profileLinks = [{ name: "Settings" }, { name: "Sign out", onClick: () => signOut() },];
```

Comment:
Removed empty onClick for vercel deplying issues

Reply:
What issue it was causing?

Reply:
There was not possible with empty onclick function.